### DISCUSSION TITLE
Student Verification Denied Several Times

### DISCUSSION THREAD

**Discussion by herwhiskers** [community/community#51811] [GitHub Education]
I recently submitted an student benefit application and followed almost all the necessary steps, including:

- verifying my school email, and set it as primary email
- providing my current location (in the campus)
- took a a photo (using my browser camera) includes: a page of my student ID (it's kind of like a pamphlet, not a card), which shows its expiring date, a translation of it, and my campus access card which indicates my school's name.

However, my application was rejected repeatedly, which left me feeling confused and disappointed. Is anything else I could try?

**Comment by samus-aran** [comment]
Hi @herwhiskers  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by herwhiskers** [reply]
Thank you! I have received these feedbacks:

- to contain my school name in the pictures I submit (though the name is in the pictures all the time)
- to use the device camera (which makes the photo not so clear)

With adjustments, these two feedback disappeared. 

However, the request for me to write my full name on the user profile is still there. **I have tried adding my full name to the profile, but I still cannot pass the verification** receiving the same feedback. (I view my full name as a privacy concern and prefer not to disclose it online, so after this unsuccessful attempt, I removed it again from my personal profile now.) 

I appreciate your help with this!

![image](https://user-images.githubusercontent.com/12695323/230708883-0051994e-1786-4089-93d2-26ed5bf43da4.png)
